### PR TITLE
Futarchy Signaling Markets for AGP Votes

### PR DESCRIPTION
**Pull request by jpkcambridge**

### CODE REVIEW

File: AGPs/AGP-FutarchySignaling.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Title: Futarchy Signaling Markets for AGP Votes
+Author: John Kelleher (@jpkcambridge)
+Status: Stage III
+Track: Finance
+Created: 2019-04-06
+---
+
+## Address of the transfer recipient
+
+TBD
```
Comment (john-light) [inline AGPs/AGP-FutarchySignaling.md]: A recipient address must be specified

File: AGPs/AGP-FutarchySignaling.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)
+
+1
+
+## Purpose of the transfer
+
+"Aragon is a fight for freedom.  Aragon empowers freedom by creating liberating tools that leverage decentralized technologies." - [AGP-0, The Aragon Manifesto](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-0.md)
+
+[Futarchy](http://mason.gmu.edu/~rhanson/futarchy.html), described as "Vote your Values, Bet Your Beliefs," offers a [collusion resistant](https://www.vitalik.ca/general/2019/04/03/collusion.html) way for organizations to make decisions. By using prediction markets, participants are incentivized to evaluate a decision based on a given metric.  For organizations aligned around given values, futarchy offers the promise of enabling well informed decisions without delegating power to the few.  Futarchy offers the potential to be a liberating tool, and advancements in decentralized technology are allowing it to come to fruition.  We seek to demonstrate and improve upon an implementation of futarchy, better understand futarchy in practice, and to educate the Aragon network and the public at large on the promise of Futarchy.
```
Comment (john-light) [inline AGPs/AGP-FutarchySignaling.md]: > to educate the Aragon network and the public at large on the promise of Futarchy

Is "futarchy" supposed to be capitalized here? It seems to be spelled lower-cased elsewhere in the doc.

File: AGPs/AGP-FutarchySignaling.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Our team has been developing an [Aragon app for futarchy decision markets](https://github.com/levelkdev/futarchy-app), as part of [this NEST proposal](https://github.com/aragon/nest/issues/93). As the first use case for the futarchy app, we're planning to build "Signaling Markets" for AGP votes using ANT price as the metric.
```
Comment (john-light) [inline AGPs/AGP-FutarchySignaling.md]:
```suggestion
Our team has been developing an [Aragon app for futarchy decision markets](https://github.com/levelkdev/futarchy-app), as part of [this Nest proposal](https://github.com/aragon/nest/issues/93). As the first use case for the futarchy app, we're planning to build "Signaling Markets" for AGP votes using ANT price as the metric.
```

File: AGPs/AGP-FutarchySignaling.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+## Recipient information
+
+Organization (if any)  
+Name: Level K
+Website: https://www.levelk.io/
+Other URL:  N/A
+
+Fill out the following information for each individual team member who will be managing funds from this transfer:
+
+Name: TBD
```
Comment (john-light) [inline AGPs/AGP-FutarchySignaling.md]: This section should be completed.

**Comment by john-light** [comment]
Please let the AGP editors know when your proposal has been finalized and you would like to submit it for final review by the Aragon Association. The deadline to finalize and submit your proposal is 16:00 April 11th. Proposals not finalized at this time will not be submitted for review.

**Comment by jpkcambridge** [comment]
@john-light Thanks for the review.  I pushed a commit which I believe addresses all comments.

Cc @lkngtn

**Comment by jpkcambridge** [comment]
@john-light @lkngtn We are ready to consider this proposal finalized.

**Comment by john-light** [comment]
A technical issue with GitHub is preventing this PR from being merged, but from an AGP Editorial perspective this AGP has been finalized and will be reviewed by the Aragon Association Board of Directors for ANV-02. We will try to resolve this technical issue as soon as possible.

**Comment by jpkcambridge** [comment]
> A technical issue with GitHub is preventing this PR from being merged, but from an AGP Editorial perspective this AGP has been finalized and will be reviewed by the Aragon Association Board of Directors for ANV-02. We will try to resolve this technical issue as soon as possible.

Glad to hear and thanks John.  Let me know if anything needs to be done on my end.

**Comment by luisivan** [comment]
@jpkcambridge  can you try signing the CLA now?

**Comment by jpkcambridge** [comment]
@luisivan Just clicked on the `Details` link next to "licencse/cla Pending - Contributo License Agreement is not signed yet." and signed the CLA with github.  Now when I click on the same link it says I have signed it.  Not sure if it solved the problem though because it still says pending?

**Comment by luisivan** [comment]
No prob, we temporarily disabled the check @jpkcambridge

**Pull request merged**